terraform/ec2/ec2.tf

```diff
@@ -0,0 +1,97 @@
+provider "aws" {
+  region = var.region
+}
+
+data "aws_s3_bucket_object" "ssh_private_key" {
```

Comment:
Is it possible to generate private key with terraform instead of maintaining a bucket?

https://registry.terraform.io/providers/hashicorp/tls/latest/docs/resources/private_key

It would be ephemereal and deleted when terraform destroy happens automatically

Reply:
i'm trying to keep the private key in case we need to log into the ec2 instance for debugging. we usually need to log into the instance to debug the collector and the image as the log of integ-test can't tell too much info to us.